app/models/emoji.rb

```diff
@@ -0,0 +1,31 @@
+class Emoji < ApplicationRecord
+  def self.all
+    self.new.all
+  end
+
+  def all
+    list_of_emojis
+  end
+
+  private
+
+  def list_of_emojis
```

Comment:
toàn bộ các hàm ở đây e chuyển về helper giúp anh, vì nó liên quan nhiều dến hiển thị ra màn hình